client/nginx.go

```diff
@@ -132,6 +132,7 @@ type Stats struct {
 	HTTPLimitRequests      HTTPLimitRequests
 	HTTPLimitConnections   HTTPLimitConnections
 	StreamLimitConnections StreamLimitConnections
+	Workers                []*Workers
```

Comment:
Should we use a map like for the other ones?

Reply:
The response is an array (updated the PR description to show). So I just sent it back as it was in the API. Happy to change it to a map but I'll have to convert it from an array. Then I can key off the id? See the example JSON for reference: 

```  {
    "id": 0,
    "pid": 13,
    "connections": {
      "accepted": 81,
      "dropped": 0,
      "active": 1,
      "idle": 0
    },
    "http": {
      "requests": {
        "total": 100,
        "current": 1
      }
    }
  },```

Reply:
Huh... this is interesting. I'm not sure if we should keep our library consistent or just expose it as it is in the API tbh

Reply:
https://github.com/nginxinc/nginx-plus-go-client/blob/main/client/nginx.go#L343 Peers already has an array so I guess it is the same?

```curl http://127.0.0.1:8000/api/8/http/upstreams/u | jq
{
  "peers": [
    {
      "id": 0,
      "server": "127.0.0.1:8081",
      "name": "127.0.0.1:8081",
      "backup": false,
      "weight": 1,
      "state": "up",
      "active": 0,
      "ssl": {
        "handshakes": 4,
        "handshakes_failed": 0,
        "session_reuses": 2
      },
      "requests": 4,
      "header_time": 4,
      "response_time": 4,
      "responses": {
        "1xx": 0,
        "2xx": 4,
        "3xx": 0,
        "4xx": 0,
        "5xx": 0,
        "codes": {
          "200": 4
        },
        "total": 4
      },
      "sent": 336,
      "received": 628,
      "fails": 0,
      "unavail": 0,
      "health_checks": {
        "checks": 0,
        "fails": 0,
        "unhealthy": 0
      },
      "downtime": 0,
      "selected": "2022-06-03T00:25:29Z"
    },
    {
      "id": 1,
      "server": "127.0.0.1:8082",
      "name": "127.0.0.1:8082",
      "backup": false,
      "weight": 1,
      "state": "up",
      "active": 0,
      "ssl": {
        "handshakes": 3,
        "handshakes_failed": 0,
        "session_reuses": 1
      },
      "requests": 3,
      "header_time": 4,
      "response_time": 4,
      "responses": {
        "1xx": 0,
        "2xx": 3,
        "3xx": 0,
        "4xx": 0,
        "5xx": 0,
        "codes": {
          "200": 3
        },
        "total": 3
      },
      "sent": 252,
      "received": 471,
      "fails": 0,
      "unavail": 0,
      "health_checks": {
        "checks": 0,
        "fails": 0,
        "unhealthy": 0
      },
      "downtime": 0,
      "selected": "2022-06-03T00:25:29Z"
    }
  ],
  "keepalive": 0,
  "zombies": 0,
  "zone": "uz"
}```

Reply:
well stream does return an array ```curl http://0.0.0.0:8080/api/9/stream/
["server_zones","limit_conns","keyvals","zone_sync","upstreams"]``` but I think that's a slightly different use case